src/platform/datapath_raw_xdp_linux.c

```diff
@@ -195,7 +810,23 @@ CxPlatDpRawRelease(
     _In_ XDP_DATAPATH* Xdp
     )
 {
-    UNREFERENCED_PARAMETER(Xdp);
+    QuicTraceLogVerbose(
+        XdpRelease,
+        "[ xdp][%p] XDP release",
+        Xdp);
+    if (CxPlatRefDecrement(&Xdp->RefCount)) {
```

Comment:
main thread and platform worker thread access this func.
Becasue `Xdp->RefCount` is 2, first call from main thread is ignored, then second call from platform worker thread.
The platform based call sometimes happens during next datapath initialization, which cause `bind()` to return EBUSY and/or causing unexpected bahavior.

Each tests in msquicplatformtest calls `CxPlatDpRawInitialize - CxPlatDpRawUnInitialize` 4 times (* # of iface) during initialization which send/recv unreachable

Reply:
hmm, this is one factor, there seems another reason

Reply:
Looks like known issue?
https://www.spinics.net/lists/xdp-newbies/msg01645.html

Reply:
Retry could fix the issue, but another issue appears.

3rd call of CxPlatDataPathInitialize hits EBUSY when CxPlatCreateUdp is called twice.
Just calling CxPlatDatapathInitialize without CxPlatCreateUdp doesn't hit the issue.
```c
int count = 3;
while (count--) {
    CxPlatDataPath Datapath(&EmptyUdpCallbacks);
    VERIFY_QUIC_SUCCESS(Datapath.GetInitStatus());
    ASSERT_NE(nullptr, Datapath.Datapath);

    CxPlatSocket Socket(Datapath);
    VERIFY_QUIC_SUCCESS(Socket.GetInitStatus());
    ASSERT_NE(nullptr, Socket.Socket);
}
```

Reply:
Fixed by overcommitting QueueCount (# of socket) to 2. default is 1 for duonic....

Reply:
That was wrong fix.

Now fixed by `AuxSocket` to fill by `INVALID_SOCKET` instead of `0`